src/app/fdctl/run/tiles/fd_verify.h

```diff
@@ -38,16 +38,25 @@ typedef struct {
   ulong       out_chunk0;
   ulong       out_wmark;
   ulong       out_chunk;
+
+  ulong       hashmap_seed;
 } fd_verify_ctx_t;
 
-#define FD_VERIFY_DEDUP_TAG_FROM_PAYLOAD_SIG(payload_sig_p) FD_LOAD( ulong, (payload_sig_p) )
+FD_FN_PURE static inline ulong
+fd_hashmap_hash( uchar const k[ 64 ],
+                 ulong seed ) {
+  ulong * kul = (ulong *)k;
+  return ( (fd_ulong_hash( seed ^ (1UL<<0) ^ kul[0] ) ^ fd_ulong_hash( seed ^ (1UL<<1) ^ kul[1] ) ) ^
+           (fd_ulong_hash( seed ^ (1UL<<2) ^ kul[2] ) ^ fd_ulong_hash( seed ^ (1UL<<3) ^ kul[3] ) ) ) ^
+         ( (fd_ulong_hash( seed ^ (1UL<<4) ^ kul[4] ) ^ fd_ulong_hash( seed ^ (1UL<<5) ^ kul[5] ) ) ^
+           (fd_ulong_hash( seed ^ (1UL<<6) ^ kul[6] ) ^ fd_ulong_hash( seed ^ (1UL<<7) ^ kul[7] ) ) ); /* tons of ILP */
+}
 
 static inline int
 fd_txn_verify( fd_verify_ctx_t * ctx,
                uchar const *     udp_payload,
                ushort const      payload_sz,
-               fd_txn_t const *  txn,
-               ulong *           opt_sig ) {
```

Comment:
This opt_sig doesn't need to be propagated anymore, so we can remove it for simplicity.